pkg/pillar/cmd/zedmanager/handleverifier.go

```diff
@@ -252,27 +260,75 @@ func handleVerifyImageStatusDelete(ctxArg interface{}, key string,
 	log.Infof("handleVerifyImageStatusDelete done for %s\n", key)
 }
 
-// check whether the cert files are installed
-func checkCertsForObject(safename string, ss *types.StorageStatus) bool {
-
-	var cidx int = 0
-
+// whether the object contains certs reference
+func containsCerts(safename string, ss *types.StorageStatus) bool {
+	cidx := 0
 	// count the number of cerificates in this object
 	if ss.SignatureKey != "" {
 		cidx++
 	}
-	for _, certUrl := range ss.CertificateChain {
-		if certUrl != "" {
+	for _, certURL := range ss.CertificateChain {
+		if certURL != "" {
 			cidx++
 		}
 	}
 	// if no cerificates, return
 	if cidx == 0 {
-		log.Infof("checkCertsForObject() for %s, no certificates configured\n",
+		log.Infof("containsCerts() for %s, no certificates configured\n",
 			safename)
-		return true
+		return false
+	}
+	return true
+}
+
+// check for cert object status
+func checkCertsStatusForObject(ctx *zedmanagerContext, uuidStr string,
```

Comment:
We repeated the code 3 times here.. Why? Can we simplify this into some utility routines?